lib/sync-wrapper.js

```diff
@@ -0,0 +1,17 @@
+module.exports = syncWrapper
+
+var resolveObject = require('./utils/resolve-object')
+
+function syncWrapper (method, docsOrIds) {
+  var defer
+
+  if (docsOrIds) {
+    defer = this[method](docsOrIds)
+  } else {
+    defer = this[method]()
+  }
```

Comment:
I think you can just pass on docsOrIds, if it's undefined, it's undefined :)

Also the methods don't return a defer, they return a promise. A defer would expose methods like `resolve` and `reject`